scripts/build-profiles/stencil-component/commands/build.js

```diff
@@ -0,0 +1,55 @@
+const { execSync } = require('child_process');
+const { existsSync } = require('fs');
+const { resolve } = require('path');
+const { info } = require('@okta/scripts.logger');
+
+exports.command = 'build';
+exports.desc = 'Builds Stencil components';
+exports.builder = {
```

Comment:
Does this mean we'll need a separate package to contain a build profile for each component type? Instead, what if we exposed multiple choices and imported the correct build profile from there?

Example in [this PR](https://github.com/okta/odyssey/blob/f3535e8806d1f91577e656b6dbdfa9954d8a5bac/scripts/tools/commands/build.js).